nullaway/src/main/java/com/uber/nullaway/fixserialization/Serializer.java

```diff
@@ -164,4 +165,29 @@ private void appendToFile(String row, Path path) {
       throw new RuntimeException("Error happened for writing at file: " + path, e);
     }
   }
+
+  /**
+   * Converts the given uri to the real path. Note, in NullAway CI tests, source files exists in
+   * memory and there is no real path leading to those files. Instead, we just serialize the path
+   * from uri as the full paths are not checked in tests.
+   *
+   * @param uri Given uri.
+   * @return Real path for the give uri.
+   */
+  @Nullable
+  public static Path pathToSourceFileFromURI(@Nullable URI uri) {
+    if (uri == null) {
+      return null;
+    }
+    if (uri.getScheme() == null) {
```

Comment:
Not a full review, but just the patch I needed to get this to run with buck
```suggestion
    if (!"file".equals(uri.getScheme())) {
```